### PR TITLE
[MIG] account_asset_batch_compute: Migration to 14.0

### PR DESCRIPTION
**Pull request by olgamarcocb**
Standard Migration

### CODE REVIEW

**Review by etobella — APPROVED** [review]

File: account_asset_batch_compute/__manifest__.py [hunk 1]
```diff
@@ -5,10 +5,10 @@
     "name": "Account Asset Batch Compute",
     "summary": """
         Add the possibility to compute assets in batch""",
-    "version": "13.0.1.0.0",
+    "version": "14.0.1.0.0",
     "license": "AGPL-3",
     "author": "ACSONE SA/NV,ForgeFlow,Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/account-financial-tools",
-    "depends": ["account_asset_management", "queue_job"],
+    "depends": ["account_asset_management", "queue_job", "account_fiscal_year"],
```
Comment (AaronHForgeFlow) [inline account_asset_batch_compute/__manifest__.py]: Why changing the dependency is needed?

Comment (olgamarcocb) [reply account_asset_batch_compute/__manifest__.py]: My bad! Thank you!

**Review by etobella — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
Code review LGTM!

**Pull request merged**